### PR TITLE
Task1 to implement models

### PR DESCRIPTION
**Pull request by Platton96**

### CODE REVIEW

File: MoneySplitter.Models/Models/FriendsOfUser.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Linq;
+
+namespace MoneySplitter.Models.Models
+{
+    public class FriendsOfUser
+    {
+        public int Id { get; set; }
+        public int UserId { get; set; }
+        public IQueryable<User> Friends { get; set; }
```
Comment (Vovanella95) [inline MoneySplitter.Models/Models/FriendsOfUser.cs]: I don't think we need such class. Just add IEnumerable<User> property to user model

File: MoneySplitter.Models/Models/Transaction.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace MoneySplitter.Models.Models
```
Comment (Vovanella95) [inline MoneySplitter.Models/Models/Transaction.cs]: Looks like you added folder Models to Models project. Better name it more informatically. For example Api, in this case we will have Models.Api namespace, it looks more nicely

File: MoneySplitter.Models/Models/Transaction.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+{
+    public class Transaction
```
Comment (Vovanella95) [inline MoneySplitter.Models/Models/Transaction.cs]: Don't forget name models like Name + Model
TransactionModel, UserModel.....

File: MoneySplitter.Models/Models/User.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+﻿namespace MoneySplitter.Models.Models
+{
+    public class User
```
Comment (Vovanella95) [inline MoneySplitter.Models/Models/User.cs]: UserModel

File: MoneySplitter.Models/Models/FriendsOfUser.cs (hunk 1 above)
Comment (Platton96) [reply MoneySplitter.Models/Models/FriendsOfUser.cs]: remove

File: MoneySplitter.Models/Models/Transaction.cs (hunk 2 above)
Comment (Platton96) [reply MoneySplitter.Models/Models/Transaction.cs]: rename Models to Entities

File: MoneySplitter.Models/Models/Transaction.cs (hunk 3 above)
Comment (Platton96) [reply MoneySplitter.Models/Models/Transaction.cs]: rename TransactionModel

File: MoneySplitter.Models/Models/User.cs (hunk 4 above)
Comment (Platton96) [reply MoneySplitter.Models/Models/User.cs]: rename User to UserModel

**Pull request merged**